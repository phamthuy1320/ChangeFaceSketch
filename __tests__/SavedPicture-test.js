import React from 'react';
import renderer from 'react-test-renderer';
import SavedPicture from '../src/screens/SavedPicture';

test('Picture Saved', () => {
  const tree = renderer.create(<SavedPicture/>).toJSON();
  expect(tree).toMatchSnapshot();
});