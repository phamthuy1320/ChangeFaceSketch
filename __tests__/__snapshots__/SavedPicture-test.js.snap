// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picture Saved 1`] = `
Array [
  <View
    style={Object {}}
  >
    <View
      style={
        Array [
          Object {
            "alignContent": "center",
            "alignItems": "center",
            "flexDirection": "row",
            "height": 50,
            "justifyContent": "space-between",
            "paddingHorizontal": 5,
          },
          Object {
            "backgroundColor": "#9f4347",
          },
        ]
      }
    >
      <Text
        allowFontScaling={false}
        onPress={[Function]}
        style={
          Array [
            Object {
              "color": "#fff",
              "fontSize": 22,
            },
            undefined,
            Object {
              "fontFamily": "anticon",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
      <Text
        style={
          Array [
            Object {
              "textAlign": "center",
            },
            Object {
              "color": "#fff",
            },
          ]
        }
      >
        PHOTO
      </Text>
      <View />
    </View>
    <View />
  </View>,
  <View
    style={
      Object {
        "alignContent": "flex-start",
        "backgroundColor": "#f1c3aa",
        "flex": 1,
        "justifyContent": "flex-start",
        "padding": 5,
      }
    }
  >
    <Image
      resizeMode="contain"
      source={
        Object {
          "uri": "https://i.pinimg.com/564x/bd/0f/4f/bd0f4f9d99dee4fd5c3ea53facc7492f.jpg",
        }
      }
      style={
        Object {
          "flex": 1,
          "height": "100%",
          "width": "100%",
        }
      }
    />
    <View
      style={
        Object {
          "alignContent": "center",
          "flexDirection": "row",
          "justifyContent": "space-around",
          "padding": 10,
        }
      }
    >
      <View>
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignContent": "center",
              "alignItems": "center",
              "aspectRatio": 1,
              "backgroundColor": "#fff",
              "borderRadius": 50,
              "height": 50,
              "justifyContent": "center",
              "opacity": 1,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "#9f4347",
                  "fontSize": 22,
                },
                undefined,
                Object {
                  "fontFamily": "Entypo",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
        <Text
          style={
            Array [
              Object {
                "textAlign": "center",
              },
              Object {
                "color": "#9f4347",
              },
            ]
          }
        >
          Change
        </Text>
      </View>
      <View>
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignContent": "center",
              "alignItems": "center",
              "aspectRatio": 1,
              "backgroundColor": "#fff",
              "borderRadius": 50,
              "height": 50,
              "justifyContent": "center",
              "opacity": 1,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "#9f4347",
                  "fontSize": 22,
                },
                undefined,
                Object {
                  "fontFamily": "Entypo",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
        <Text
          style={
            Array [
              Object {
                "textAlign": "center",
              },
              Object {
                "color": "#9f4347",
              },
            ]
          }
        >
          Main
        </Text>
      </View>
    </View>
  </View>,
]
`;
